src/pages/RequestCallPage.js

```diff
@@ -280,68 +297,80 @@ class RequestCallPage extends Component {
+                                                onBlur={this.validatePhoneInput}
+                                                onChangeText={phoneNumber => this.setState({phoneNumber})}
+                                            />
+                                            <TextInput
+                                                label={this.props.translate('requestCallPage.extension')}
+                                                keyboardType={CONST.KEYBOARD_TYPE.PHONE_PAD}
+                                                autoCompleteType="off"
+                                                autoCorrect={false}
+                                                value={this.state.phoneExtension}
+                                                placeholder="100"
+                                                errorText={this.state.phoneExtensionError}
+                                                onBlur={this.validatePhoneExtensionInput}
+                                                onChangeText={phoneExtension => this.setState({phoneExtension})}
+                                                containerStyles={[styles.mt4]}
+                                            />
+                                            <Text style={[styles.textMicroSupporting, styles.mt4]}>{this.getWaitTimeMessage()}</Text>
+                                        </Section>
+                                    </FormElement>
+                                </ScrollView>
+                                <FixedFooter>
+                                    {isBlockedFromConcierge && (
+                                        <View style={[styles.flexRow, styles.alignItemsCenter, styles.mb3]}>
+                                            <Icon src={Expensicons.Exclamation} fill={colors.yellow} />
+                                            <Text style={[styles.mutedTextLabel, styles.ml2, styles.flex1]}>{this.props.translate('requestCallPage.blockedFromConcierge')}</Text>
+                                        </View>
+                                    )}
+                                    {!_.isEmpty(this.props.requestCallForm.error) && (
+                                        <Text style={styles.formError}>
+                                            {this.props.requestCallForm.error}
+                                        </Text>
+                                    )}
+                                    <Button
+                                        success
+                                        pressOnEnter
+                                        onPress={this.onSubmit}
+                                        style={[styles.w100]}
+                                        text={this.props.translate('requestCallPage.callMe')}
+                                        isLoading={this.props.requestCallForm.loading}
+                                        isDisabled={isBlockedFromConcierge}
+                                    />
```

Comment:
To implement offline UX pattern C we need to disable this button, prevent the form from submitting, and show the user that they are offline.

Reply:
Ah yes. Using `FormAlertWithSubmitButton` to fix this. Thanks!